src/definitions.jl

```diff
@@ -344,6 +344,17 @@ plan_irfft
 
 ##############################################################################
 
+"""
+	fftshift!(dest, src, [dim])
+
+Nonallocating version of [`fftshift`](@ref). Stores the result of the shift of the `src` array into the `dest` array.
+"""
+function fftshift!(dest, src, dim = 1:ndims(src))
+	@assert size(dest)==size(src)
```

Comment:
Also, this shouldn't use an `@assert`, it should throw a `DimensionMismatch` exception.

But I don't think you need to check anything at all here, since `circshift!` already checks this.

Reply:
Sorry, I thought my vim automatically expands the tabs, but I actually forgot this setup in vimrc file.